tests/integration/security/healthcheck/main_test.go

```diff
@@ -35,8 +34,6 @@ var (
 func TestMain(m *testing.M) {
 	framework.NewSuite("mtls_healthcheck", m).
 		RequireEnvironment(environment.Kube).
-		Label(label.CustomSetup).
-		SetupOnEnv(environment.Kube, istio.Setup(&ist, setupConfig)).
```

Comment:
if you are removing this it doesn't need to be in its own package, so its a bit faster by reusing istio setup

Reply:
Done. I end up moving them to the same istio setup.